src/Workspaces/Remote/Core/Storage/RemotePersistentStorageLocationService.cs

```diff
@@ -8,8 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.Remote.Storage
 {
-    [ExportWorkspaceService(typeof(IPersistentStorageLocationService)), Shared]
-    [Export(typeof(RemotePersistentStorageLocationService))]
+    [ExportWorkspaceService(typeof(IPersistentStorageLocationService), layer: SolutionService.WorkspaceKind), Shared]
```

Comment:
not sure. for now, I think we can leave it as it is since this kind is not shared out side of this dll.